docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-metrics-ui.asciidoc

```diff
@@ -0,0 +1,68 @@
+[role="xpack"]
+[[ootb-ml-jobs-metrics-ui]]
+= Metrics {anomaly-detect} configurations
+++++
+<titleabbrev>Metrics</titleabbrev>
+++++
+
+// tag::metrics-jobs[]
+These {anomaly-jobs} can be created in the
+{kibana-ref}/xpack-infra.html[Metrics app] in {kib}. When you create one of 
+these jobs, you need to select a field that splits the data to establish 
+separate baselines for the job. This is called a partion field. Each value of 
+the partition field is modeled individually which enables more independent 
+anomaly scoring.
+
+When you select partition field for a job, take into account field cardinality; 
+the number of different values that the field contains. If the partition field 
+has more than 1000 distinct values per job, you are advised that there might be 
+high memory usage.
+
+The jobs below detect anomalous memory and network behavior on hosts and 
+Kubernetes pods. For more details, see the {dfeed} and job definitions in the 
+`metrics_ui_*` folders in https://github.com/elastic/kibana/tree/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules[GitHub].
+
+
+hosts_memory_usage::
+
+* For memory usage via the Metrics UI.
```

Comment:
> For memory usage via the Metrics UI.

I know we used the same format for the Logs jobs, but I'm not sure what the "via the xxx UI" really means here.  Does it mean you're viewing this in the Metrics UI (in which case I think that's redundant, since that true of all these)?  Or does it mean that it's limited to data that you've already configured for use in the Metrics app (in which case it's clearer to say something like "For memory usage data in the {metrics-app}", IMO)?

Reply:
Thanks! I amended the text accordingly. Will do the same for Logs jobs.